CMakeSettings.json

```diff
@@ -0,0 +1,51 @@
+﻿{
+    "configurations": [
+        {
+            "name": "nostd-x64-Debug",
+            "generator": "Ninja",
+            "configurationType": "Debug",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "buildRoot": "${projectDir}\\out\\vs2019\\${name}",
+            "installRoot": "${projectDir}\\out\\vs2019\\${name}\\install",
```

Comment:
Does this only make sense for VS2019, or for previous versions too?

Reply:
Unfortunately the proper CMake support was only added to recent update of Visual Studio 2019. This file is unused by older versions of Visual Studio. I can check if it works for Visual Studio code.. Most likely it should also work for Visual Studio Code with extension.